### PR TITLE
APEXCORE-453 decreased the heartbeat interval in the DelayOperator unit test because otherwise the committed window call may occur much later than expected, hence missing the failure simulation

### PR DESCRIPTION
**Pull request by davidyan74**
@PramodSSImmaneni  please review and merge

### CODE REVIEW

**Comment by sandeepdeshmukh** [comment]
I verified this fix. Tried it 10 times and the test case passed all 10 times.

**Comment by davidyan74** [comment]
@sandeepdeshmukh thanks for verifying this fix.

File: engine/src/test/java/com/datatorrent/stram/plan/logical/DelayOperatorTest.java [hunk 1]
```diff
@@ -233,8 +233,10 @@ public void beginWindow(long windowId)
     {
       if (simulateFailureWindows > 0 && ((simulateFailureAfterCommit && committed) || !simulateFailureAfterCommit) &&
           !failureSimulated) {
+        LOG.info("FailableFibonacciOperator beginWindow {} {} {}", windowId, windowCount, simulateFailureWindows);
```
Comment (tweise) [inline engine/src/test/java/com/datatorrent/stram/plan/logical/DelayOperatorTest.java]: Why all this INFO level logging? I think that can be removed or use DEBUG.

**Comment by davidyan74** [comment]
@tweise  done

**Pull request merged**